ui/src/ensembler/components/table/ListRouterVersionsForEnsemblerTable.js

```diff
@@ -51,6 +51,9 @@ export const ListRouterVersionsForEnsemblerTable = ({
   useEffect(() => {
     if (results.activeItems.length > 0){
       setEnsemblerUsedByActiveRouterVersion(true)
+      // If there is an active router version using this ensembler, it is definitely also a current router version using
```

Comment:
This is not necessarily true, right? A router's current version is only updated after a deployment succeeds. In the scenario where a router is being updated to a new version, say, from v4 to v5, it could be the case that v5 refers to the ensembler (which is an active version according to the logic above) but v4 (the current version) doesn't.

Reply:
Hmmm 🤔 that makes sense. Okay let me take a look at how to iron out all these conditional cases again. Thanks for pointing this out!

Reply:
Okay, just to share an update on this, we've decided to display *both* current and active router versions if an ensembler is associated with router versions of both types. In addition, if any of the associated router versions are *both* current and active, we'll only display that router version in the current router version table. This requires some additional refactoring which I will annotate shortly.